ci-operator/step-registry/ipi/conf/nutanix/minimal-permission/ipi-conf-nutanix-minimal-permission-ref.yaml

```diff
@@ -0,0 +1,17 @@
+ref:
+  as: ipi-conf-nutanix-minimal-permission
+  from_image:
+    namespace: ocp
+    name: "4.12"
+    tag: upi-installer
+  commands: ipi-conf-nutanix-minimal-permission-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  credentials:
+  - namespace: test-credentials
+    name: nutanix-pc
```

Comment:
This credential is from our QE's vault?

Reply:
yes

Reply:
Can you help point me where to define it? I did not find it.

Reply:
It's in https://vault.ci.openshift.org/ui/vault/secrets/kv/show/selfservice/openshift-qe/nutanix-qe-dis/secrets.sh

Reply:
For `nutanix-qe-dis` vault,  all the items are under ci/cluster-secrets-nutanix-qe-dis, I did not see where "test-credentials/nutanix-pc" are defined in that vault.

Reply:
Got it, I copied it from step ipi-conf-nutanix-context, actually it's not needed, it will use secrets in CLUSTER_PROFILE_DIR, I'll remove it

`if [[ -f ${CLUSTER_PROFILE_DIR}/secrets.sh ]]; then
  NUTANIX_AUTH_PATH=${CLUSTER_PROFILE_DIR}/secrets.sh
else
  NUTANIX_AUTH_PATH=/var/run/vault/nutanix/secrets.sh
fi`

Reply:
Fixed